lightningd/invoice.c

```diff
@@ -298,7 +299,7 @@ static void json_add_invoices(struct json_result *response,
 		json_object_start(response, NULL);
 		json_add_string(response, "label", i->label);
 		json_add_hex(response, "rhash", &i->rhash, sizeof(i->rhash));
-		json_add_u64(response, "msatoshi", i->msatoshi);
+		json_add_u64(response, "msatoshi", *i->msatoshi);
 		json_add_bool(response, "complete", i->state == PAID);
```

Comment:
This needs to be conditional, otherwise we segv when i->msatoshi is NULL.

Reply:
toink! Yes, will fix.